.github/sync-repo-settings.yaml

```diff
@@ -0,0 +1,20 @@
+rebaseMergeAllowed: true
+squashMergeAllowed: true
+mergeCommitAllowed: false
+branchProtectionRules:
+- pattern: master
+  isAdminEnforced: true
+  requiredStatusCheckContexts:
+    - 'ci/circleci: build'
+    - 'ci/circleci: build_cloud_android_examples'
+    - 'ci/circleci: build_cloud_examples'
+    - 'ci/circleci: dockerize'
+    - 'ci/circleci: test_example'
```

Comment:
I'm not sure if there is showcase integration test in place for this. May be we should consider removing this check.

Adding Noah to chime in.

Reply:
I'm not sure why it's failing, but realistically this config is just matching what's already set up in branch protection.

Reply:
Hey there! So, a few of these checks are going to fail now because they relied on GCP projects that have been shutdown. IIRC the showcase tests were among those.